src/lib/authorization.middleware.ts

```diff
@@ -0,0 +1,41 @@
+import { NextFunction, Request, Response } from "express";
+import { errorResponse } from "./responses/error.response";
+import { developmentAuthorize } from "./development.authorize";
+import { oktaAuthorization } from "./okta.authorize";
+
+/**
+ * @description - Authorization middleware to filter invalid requests.
```

Comment:
I'm not a fan of documenting private functions in our code unless they're absolutely necessary. esp. now with typescript we now best what the parameters are by looking at the function signature than the doc. It's more code to maintain :) can you remove this description and the other ones in the code?